<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QTabWidget" name="mainQTabWidget">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="BuildQTab">
      <attribute name="title">
       <string>Build</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>  Modules:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="modulesQTree">
         <column>
          <property name="text">
           <string>Systems</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Namespace:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="moduleQName">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>+ Side:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="moduleQSide">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>cn</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>lf</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>rt</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>up</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>dw</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>fr</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>bc</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="referenceQButton">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Reference</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="importQButton">
           <property name="text">
            <string>Import</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="referenceEditorQButton">
         <property name="text">
          <string>Reference Editor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Utilities:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="templateEditQButton">
            <property name="text">
             <string>Template Edit</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="mirrorQButton">
              <property name="text">
               <string>Mirror YZ</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="mirrorQCheck">
              <property name="maximumSize">
               <size>
                <width>65</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Connect</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="templateGetPoseQButton">
              <property name="text">
               <string>Get Pose</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="templateWritePoseQButton">
              <property name="text">
               <string>Write Pose File</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="templateSetPoseQButton">
              <property name="text">
               <string>Set Pose</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="templateSetPoseQCheck">
              <property name="maximumSize">
               <size>
                <width>65</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>From file</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Compound:</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPlainTextEdit" name="charDocQPText">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="modCompPathQLine">
            <property name="placeholderText">
             <string>Modules Compound Path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveModCompQButton">
            <property name="text">
             <string>Save Modules Compound</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="breakRefsQButton">
         <property name="text">
          <string>Break References</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="DevelopQTab">
      <attribute name="title">
       <string>Develop</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QListWidget" name="methodsQList"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Config:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="systemQVBoxLayout"/>
       </item>
       <item>
        <widget class="QPushButton" name="systemQButton">
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Utilities:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="namespaceEditorQButton">
            <property name="text">
             <string>Namespace Editor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hierarchizeQButton">
            <property name="text">
             <string>Hierarchize</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hierarchyToogleQButton">
            <property name="text">
             <string>Hierarchy Toogle</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
